packages/atomic/src/components/search/atomic-refine-modal/atomic-refine-modal.tsx

```diff
@@ -209,7 +209,8 @@ export class AtomicRefineModal implements InitializableComponent {
       return;
     }
     const amountToCollapseNext =
-      this.collapseFacetsAfter - this.bindings.store.getFacetElements().length;
+      this.collapseFacetsAfter -
+      this.bindings.store.getVisibleFacetElementsLength();
     const automaticFacets = isAutomaticFacets
       ? this.automaticFacetBuilderState.automaticFacets.map((facet, index) => {
           const isCollapsed = index >= amountToCollapseNext;
```

Comment:
Out of scope for this PR, but I haven't noticed until now. Is it possible currently to style automatic facet parts?

For example, it's currently possible with CSS to change the padding of facet values, even when they're in a refine modal, by using `::part(SOMETHING)`. Does this also work with automatic facets?

Part of why I'm asking, is because I vaguely recall now that this is the reason why the regular facets render a `slot` element.

Reply:
it is because of the shadow dom ! I am working on a better implementation